packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -92,11 +102,14 @@ export const getL2TestnetNetworkConfig = (): NetworkConfig => {
 export const ERC20TokenName = 'Test Arbitrum Token'
 export const ERC20TokenSymbol = 'TESTARB'
 export const ERC20TokenDecimals = 18
-export const invalidTokenAddress = '0x0000000000000000000000000000000000000000'
+export const invalidTokenAddress = utils.computeAddress(utils.randomBytes(32))
```

Comment:
with custom fee token WETH is deployed to zero address for some reason, we use a random address instead